Program.cs

```diff
@@ -1,7 +1,66 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿// --------------------------------------------------------------------------------------------
+// Training ~ A training program for new interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ------------------------------------------------------------------------
+// Program.cs
+// Implement a Stack<T> class using arrays as the underlying data structure.
+// The Stack<T> should start with an  initial capacity of 4 and double its capacity when needed.
+// Use the template shown below for implementation. Throw exceptions when necessary. 
+// class TStack<T> {
+//    public void Push (T a) { }
+//    public T Pop () { }
+//    public T Peek () { }
+//    public bool IsEmpty { get; }
+// }
+// InvalidOperationException: This exception should be thrown when attempting to pop or peek an empty stack. 
+// --------------------------------------------------------------------------------------------
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) { }
+   }
+   #region TStack ------------------------------------------------------------------------------
+   /// <summary>TSatck</summary>
+   /// <typeparam name="T"></typeparam>
+   public class TStack<T> {
+      private int mCount, mCapacity;
+      private T[] elements;
+
+      public TStack () {
+         mCapacity = 4; mCount = 0; elements = new T[4];
+      }

```

Comment:
Make the changes as suggested in the previous pull request (MyList<T> assignment)